ebusd-2.x.x/de/vaillant/08.recov.csv

```diff
@@ -0,0 +1,38 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
```

Comment:
2.x.x is only kept for ease of accessing older files, so please remove it from the PR
same applies to the EN file in that folder